src/platform/functions/index.js

```diff
@@ -68,14 +70,28 @@ const scrapeCityEvents = async (city) => {
             : parseInt(followers.innerText, 10);
         }
 
+        if (eventDate) {
+          if (eventDate.innerText.indexOf('+') !== -1) {
+            parsedDate = eventDate.innerText.substring(0, eventDate.innerText.indexOf('+') - 1);
+          } else {
+            parsedDate = eventDate.innerText;
+          }
+        }
+
+        if (imgUrl) {
+          if (!imgUrl.getAttribute('src').includes('data:image/gif;base64')) {
```

Comment:
magical

Reply:
haha :) I think its something to do with the ternary operators and storing the string in a variable?